core/api.go

```diff
@@ -189,6 +189,14 @@ func (a *api) Start() {
 			cafes.POST("/messages", a.checkCafeMessages)
 		}
 
+		tokens := v0.Group("/tokens")
+		{
+			tokens.POST("", a.createTokens)
+			tokens.GET("", a.lsTokens)
+			tokens.GET("/:id", a.compareTokens)
```

Comment:
Does this seems like a good RESTful approach here? Use get and delete on the token-based resource?